python/ray/autoscaler/aws/node_provider.py

```diff
@@ -45,10 +48,20 @@ def __init__(self, provider_config, cluster_name):
         NodeProvider.__init__(self, provider_config, cluster_name)
         self.cache_stopped_nodes = provider_config.get("cache_stopped_nodes",
                                                        True)
+        try:
+            aws_credentials = (
+                provider_config["extra_config"]["aws_credentials"])
+        except KeyError:
+            aws_credentials = None
```

Comment:
same as above; avoid doing this KeyError wrapping when you can use `.get`

Reply:
Done.